src/client/__tests__/e2e/pages/tile.component.ts

```diff
@@ -94,4 +97,42 @@ export class TileComponent {
     await textElement.clear()
     await textElement.sendKeys(notional)
   }
+
+  async initiateRFQ(initiateRFQ: string) {
```

Comment:
I don't really understand why this function takes a param.. what are the possible values for `initiateRFQ`?

Reply:
removed